jsk_unitree_robot/cross/repos/ros1_dependencies.repos

```diff
@@ -230,3 +230,17 @@ repositories:
   postfix:
     type: tar
     url: http://mirror.postfix.jp/postfix-release/official/postfix-3.4.3.tar.gz
+  libfcl-dev:
```

Comment:
I adopted this idea, but on the ohter hand, you can install an old version of `geometric_shape`, Currently we do not have clearner way of doing this, because the list of repositories are generated with `rosinstall_generator` (https://github.com/jsk-ros-pkg/jsk_robot/blob/93919243019ae06be43874022062abbeee00349e/jsk_unitree_robot/cross/build_ros1.sh#L65), 

may be we added something like https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_unitree_robot/cross/build_ros1.sh#L34